guides/resource-provisioning/servicebus-queues.md

```diff
@@ -0,0 +1,138 @@
+# Azure Service Bus Queues
+
+This guide describes how to add Azure Service Bus queues to a microservice running on the FFC Platform Kubernetes cluster.
+
+## Create a microservice Managed Identity
+
+If not alreay configured, [add an Azure Managed Identity to your microservice](managed-identity.md).
+
+## Request creation of Service Bus queues
+
+For each queue required by your microservices, you will need:
+* A Service Bus queue for the main branch deploy
+* A Service Bus queue for each of the developers on the workstream to use for local development
+
+Queues should follow the following naming convention:
+
+```
+ffc-<workstream>-<identifier>-<purpose>
+```
+
+where `<purpose>` either denotes the main branch deploy queue e.g. `ffc-demo-payment-dev`, or the initials of a developer for the local development queues e.g. `ffc-demo-payment-jw`.
+
+Request the creation of the required queues through your usual Cloud Services support channel.
+
+## Update Managed Identity permissions
+
+For each main branch queue created, read and/or write permissions should be added to the Managed Identites of the microservices that will be using the queues.
+
+Request Cloud Services to add the relevant permissions to the Managed Identities.
+
+Permissions do not need to be added to Managed Identities for the local development queues.
+
+## PR queue provisioning
+
+Queues can be automatically provisioned for each microservice PR by the Jenkins CI pipleine to ensure encapsulation of infrastructure.
+
+Create a `provision.azure.yaml` file in the root directory of your microservice, and populate with the queues that need provisioning:
+
+```
+resources:
+  queues:
+    - name: <identifier>
+```
+
+where the `<identifier>` relates to the part of the queue name described above. For example for the queue `ffc-demo-payment-dev`, `<identifier>` would be replaced with `payment`. Add a `name` entry for each required queue.
+
+## Update local development environment
+
+Add the following environment variables to the `.env` file in the root of the microservice to use the developer Service Bus queues during local development:
```

Comment:
Or alternatively add them to say `.bashrc` file to avoid duplication across services consuming the same things.

Reply:
Updated language to reflect different approaches